ros/src/waypoint_updater/waypoint_updater.py

```diff
@@ -3,8 +3,10 @@
 import rospy
 from geometry_msgs.msg import PoseStamped
 from styx_msgs.msg import Lane, Waypoint
+from std_msgs.msg import Int32
 
 import math
+import tf
```

Comment:
(nitpicking)
I'm not 100% sure about this, but I think we should only import the module we need instead of the whole `tf` so the process doesn't have to load the whole module into memory.
```
from tf.transformations import euler_from_quaternion
```